github/release_notes/renderer.py

```diff
@@ -210,7 +210,15 @@ def _to_bullet_points(
         bullet_points = list()
         for rls_note_obj in rls_note_objs:
             for i, rls_note_line in enumerate(rls_note_obj.text.splitlines()):
-                rls_note_line = _.trim(rls_note_line)
+                # trim '*' or '-' bullet points
+                rls_note_line = _\
+                    .chain(rls_note_line)\
+                    .trim()\
+                    .reg_exp_replace(r'^\* ', '')\
```

Comment:
it would be easier to read this by using lstrip('*') / lstrip('- '); you could even merge this into one single lstrip('*- ')

Reply:
```
>>> '  * test'.lstrip('* ')
'test'
```
OK

```
> '  *test'.lstrip('* ')
'test'
```
NOT OK

also the unit tests would fail

Reply:
i see